src/content/ui/adaptive-responsive/best-practices.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Best practices for app design
+description: >
+  Learn the best practices for creating >
+  a responsive and an adaptive app.
+short-title: Best practices
+---
+
+The best practices on this page cover,
+in particular, large screen devices,
+but can equally apply to any Flutter app.
+Flutter defines [large screens][] as tablets,
+foldables, and ChromeOS devices running Android.
+
+:::secondary Why do large screens matter, in particular?
+Demand for large screens continues to increase.
+As of January 2024,
+more than [270 million active large screen][large screens]
+and foldable devices run on Android.
+When your app supports large screens,
+it also receives the following benefits:
+
+* Optimizing your app to fill the screen improves
+  your app's user engagement metrics.
+* Increases your app's visibility in the Play Store.
+  Recent [Play Store updates][] show ratings by
+  device type and indicates when an app lacks
+  large screen support. 
```

Comment:
a third bulled around ensuring your app meets iPadOS submission guidelines and is accepted in the app store. https://developer.apple.com/ipados/submit/